front/se_project/src/components/logout.jsx

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+
+// class Logout extends React.Component {
+//     componentDidMount() { 
+//         localStorage.removeItem('token');
+//         window.location = '/'
+//     }
+
+//     render () {
+//         return null;
+//     }
+// }
+// export default Logout;
```

Comment:
that's ok. we should test it after merging!

Reply:
yes